Create Google Summer of Code 2023 webpage (#1225)
xsf/xmpp.org: content/community/gsoc-2023.de.md

---

> Title: XSF und Google Summer of Code 2023

Reviewer: ```suggestion
title: XSF und Google Summer of Code 2023
```

---

![XSF and GSoC 2022 Logo](/images/logos/GSoC_2022_Logo.png)

_Zur Kenntnis: Die Hauptkommunikation zu dem Thema findet auf englisch statt und nur die englischen Informationen gelten als offiziell._

Die XMPP Standards Foundation (XSF) plant, als gastgebende Organisation am [Google Summer of Code 2023](https://summerofcode.withgoogle.com/programs/2023) teilzunehmen.

> Wenn Sie daran interessiert sind, als Student (oder einfach nur als Open-Source-Neuling), Mentor oder XMPP-Projekt teilzunehmen, ist dies die Seite, mit der du beginnen kannst!

Reviewer: ```suggestion
Wenn du daran interessiert bist, als Student (oder einfach nur als Open-Source-Neuling), Mentor oder XMPP-Projekt teilzunehmen, ist dies die Seite, mit der du beginnen kannst!
```

> Wenn du dir vorstellen kannst, während des Sommers in Teilzeit (oder sogar Vollzeit) an einem Open-Source-Softwareprojekt zu arbeiten, dann sollten Sie sich überlegen, an einem XMPP-Kommunikationsprojekt mitzuwirken und am Google Summer of Code 2022 teilzunehmen!

Reviewer: ```suggestion
Wenn du dir vorstellen kannst, während des Sommers in Teilzeit (oder sogar Vollzeit) an einem Open-Source-Softwareprojekt zu arbeiten, dann solltest du dir überlegen, an einem XMPP-Kommunikationsprojekt mitzuwirken und am Google Summer of Code 2022 teilzunehmen!
```

### Teilnehmende Projekte

**Die XSF hat sich noch nicht beworben und wurde auch noch nicht angenommen!**

Wird angekündigt - siehe das Wiki unten

### Herzlich Willkommen neue Mitwirkenden!

Unsere neuen Mitwirkenden werden noch bekannt gegeben!

Fühlt euch frei, die Nachricht über unsere Kanäle unten zu verbreiten!

### Ressourcen und Einstiegsmöglichkeiten

Wenn ihr ein interessierter Beitragender oder ein interesiertes Projekt seid, solltet ihr ersten beiden Links unbedingt lesen! Eine frühzeitige Einbindung und Kommunikation mit uns ist für eine fruchtbare Zusammenarbeit unerlässlich.

- Lies die Ressourcen auf [Googles Website] (https://summerofcode.withgoogle.com/help) und prüfe, ob dies etwas für dich und Ihre Pläne in diesem Jahr ist.
- Wirf einen ersten Blick auf unsere XMPP-Projektdetailseite im [XSF-Wiki](https://wiki.xmpp.org/web/Google_Summer_of_Code_2023).
- Du kannst uns einfach [per Web-Chat](https://xmpp.org/chat#converse/room?jid=gsoc@muc.xmpp.org) erreichen oder die ["Getting started with XMPP"](https://xmpp.org/getting-started/) Seite durchsehen und uns über unseren [öffentlichen Chat](xmpp:gsoc@muc.xmpp.org?join) erreichen.

> Schauen dir auch unsere Medienkanäle an!

Reviewer: ```suggestion
Schaue dir auch unsere Medienkanäle an!
```
